fpr/templates/fpr/fpcommand/list.html

```diff
@@ -1,15 +1,14 @@
 {% extends "fpr/app_layout.html" %}
+{% load i18n %}
 {% load staticfiles %}
 
-{% block title %}Archivematica FPR Server - Format Policy Commands{% endblock title %}
-
-{% block page_title %}
-Format Policy Command Information
-{% endblock page_title %}
+{% block title %}{{ block.super }}Archivematica FPR Server - {% trans "Format policy commands" %}{% endblock title %}
```

Comment:
This has both `{{block.super}}` and `Archivematica FPR Server` - is that an error?